lib/_core_test.ts

```diff
@@ -44,6 +45,16 @@ Deno.test("(get/set/delete)OAuthSession() work interchangeably", async () => {
   assertEquals(await getOAuthSession(id), null);
 });
 
+Deno.test("setTokens() applies key expiry", async () => {
+  const sessionId = crypto.randomUUID();
+  const oauthSession = randomOAuthSession();
+  await setOAuthSession(sessionId, oauthSession, { expireIn: 1_000 });
+
+  assertEquals(await getOAuthSession(sessionId), oauthSession);
+  await delay(10_000);
+  assertEquals(await getOAuthSession(sessionId), null);
+});
+
```

Comment:
I'm surprised this simple test is failing. Am I missing something, @kt3k and @losfair?

Reply:
Could it have to do with [this](https://docs.deno.com/kv/manual/key_expiration#caveats)?

Reply:
In the deno [runtime tests](https://github.com/denoland/deno/blob/365d1ac7c2c6e81cbc32135ed6a099f3caaa6110/cli/tests/unit/kv_test.ts#L1878), they manually stop/start the DB to trigger immediate cleanup of expired keys

Reply:
Good catch, @mitchwadair! Yes, database stop/start is not ideal in our case. We run our tests in-memory to avoid the need for cleanup. Though, I guess it wouldn't be too hard to move away from that... @losfair, is in-memory expiration supported?

Reply:
@iuioiua Yes, expiration is handled in the same way for in-memory and persistent databases - but since the cleanup interval is long (~1 minute) it isn't practical to depend on periodic cleanup in a test.

Currently I think the best approach is to open a persistent database and re-open it to trigger an immediate cleanup...

Reply:
I've opted with not testing OAuth session expiration as it'd interfere too much with the current KV setup of this module.